### PR TITLE
[CMS-1666c] Upgrade directory-components to version 20

### PR DESCRIPTION
**Pull request by ababic**
Also pins `directory-constants` to `18.0.0`

### CODE REVIEW

File: core/sass/pages/_landing-page.scss [hunk 1]
```diff
@@ -77,7 +77,7 @@ body.webp-enabled {
 
 .landing-page {
 
-  .card-grid > * > .card {
+  .flex-grid > * > .card {
```
Comment (eadpearce) [inline core/sass/pages/_landing-page.scss]: you can remove this rule altogether as it's provided by directory-components

File: finance/templates/finance/trade_finance.html [hunk 2]
```diff
@@ -16,8 +16,9 @@
 {% block content %}
 <section id="get-finance-hero" class="get-finance-hero hero padding-top-0 padding-bottom-30" style="background-image: url({{ page.hero_image.url }})">
   <div class="container">
-		{% url 'get-finance' as get_finance_url %}
-		{% breadcrumbs left_url='/' left_label='Great.gov.uk' middle_url=get_finance_url middle_label='UKEF' right_label='Trade Finance' %}
+
+    {% breadcrumbs 'Trade Finance' %}<a href="/">Great.gov.uk</a><a href="{% url 'get-finance' %}">UKEF</a>{% endbreadcrumbs %}
```
Comment (eadpearce) [inline finance/templates/finance/trade_finance.html]: can you move these `a` tags to separate lines? makes it a bit easier to read

Comment (ababic) [reply finance/templates/finance/trade_finance.html]: sure thing :)

File: core/sass/pages/_landing-page.scss (hunk 1 above)
Comment (ababic) [reply core/sass/pages/_landing-page.scss]: Removed in eff9e7a :)

File: finance/templates/finance/trade_finance.html (hunk 2 above)
Comment (ababic) [reply finance/templates/finance/trade_finance.html]: Sorted in 6393220

**Pull request merged**